iot_config/Cargo.toml

```diff
@@ -47,6 +48,8 @@ triggered = { workspace = true }
 task-manager = { path = "../task_manager" }
 humantime-serde = { workspace = true }
 custom-tracing = { path = "../custom_tracing", features = ["grpc"] }
+solana = { path = "../solana" }
+solana-sdk = { workspace = true }
```

Comment:
We're trying to keep/move all solana deps into the `solana` workspace. Can we move/re-export whatever is needed from there?